doc/showvars.rst

```diff
@@ -0,0 +1,46 @@
+..
+  Copyright (C) 2018  Red Hat, Inc.
```

Comment:
Do you work for Red Hat? If not, it's probably better to put your own copyright here...

Reply:
Good call, I didn't look at the headers I copied really.  I'll get that fixed.